openeo/result.py

```diff
@@ -0,0 +1,76 @@
+from pathlib import Path
```

Comment:
I think this result.py file should be under `openeo.rest` instead of top level because it is rest specific